docs/source/Mux.rst

```diff
@@ -19,63 +19,63 @@ in reality it follows the way people are used to define dependencies,
 therefor it's very simple to use and clear even in complex cases.
 
 The best explanation comes usually from examples, so feel free to scroll down
-to `yaml_to_mux plugin`_ section, which uses the default mux plugin to feed
-the Mux.
+to `yaml_to_mux plugin`_ section, which uses the default variants plugin to feed
+the Variants.
 
 
-Mux internals
--------------
+Varianter internals
+-------------------
 
-The ``Mux`` is a core part of avocado and one can see it as a ``multiplexed``
+The ``Varianter`` is a core part of avocado and one can see it as a ``multiplexed``
 database, which contains key/value pairs associated to given paths and
 as we are talking about a tree of those, we call the paths ``Nodes``.
 
-Mux allows iterating through all possible combinations which are stored in
-the database, which is called ``multiplexation``. Mux yields ``variants``,
+Varianter allows iterating through all possible combinations which are stored in
+the database, which is called ``multiplexation``. Varianter yields ``variants``,
 which are lists of leaf nodes with their values, which are then processed
 into ``AvocadoParams``. Those params are available in tests as
 ``self.params`` and one can query for the current parameters::
 
     self.params.get(key="my_key", path="/some/location/*",
                     default="default_value")
 
-Let's get back to Mux for a while. As mentioned earlier, it's a database
+Let's get back to Variants for a while. As mentioned earlier, it's a database
 which allows storing multiple variants of test parameters. To fill the
 database, you can use several commands.
 
 1. ``--mux-inject`` - injects directly [path:]key:node values from the
    cmdline (see ``avocado multiplex -h``)
-2. ``yaml_to_mux plugin`` - allows parsing ``yaml`` files into the Mux
+2. ``yaml_to_mux plugin`` - allows parsing ``yaml`` files into the Variants
    database (see `yaml_to_mux plugin`_)
-3. Custom plugin using the simple ``Mux`` API (see `mux_api`_)
+3. Custom plugin using the simple ``Variants`` API (see `variants_api`_)
 
 
-.. _mux_api:
+.. _variants_api:
 
-Mux API
--------
+Varianter API
```

Comment:
yes, it's not yet pluggable, therefor it's still a single implementation depending on yaml.